process/economics/economicsData.go

```diff
@@ -291,14 +304,14 @@ func (ed *EconomicsData) NumNodes() uint32 {
 	return ed.numNodes
 }
 
-// AuctionEnabled returns whether full auction process is enabled
-func (ed *EconomicsData) AuctionEnabled() bool {
-	return ed.auctionEnabled
+// AuctionEnableNonce returns whether full auction process is enabled
```

Comment:
maybe `// AuctionEnableNonce returns the nonce that will trigger auction beginning`

Reply:
done.